documentation/docs/release-notes/3.1.0.md

```diff
@@ -0,0 +1,170 @@
+
+    **Response Length**: measures the size of the response payload in bytes, helping identify large result sets that may impact performance.
+
+=== "Lock statistics metrics"
+    New metrics for detecting lock contention issues:
+
+    - **Locks Global Acquire Count Read/Write Shared**: tracks how often global locks are used, helping to spot delays caused by multiple processes accessing the same data.
+    - **Locks Database Acquire Count Read Shared**: tracks how often read locks are used in the database.
+    - **Locks Database Acquire Wait Count Read Shared**: counts the times a database read lock request had to wait. High values may signal delays due to contention.
+    - **Locks Database Time Acquiring Micros Read Shared**: captures the time spent waiting for database read locks, critical for performance analysis.
+    - **Locks Collection Acquire Count Read Shared**: tracks how often read locks are used at the collection level.
+    - **Full Scan (COLLSCAN)**: tracks instances where the entire dataset is scanned, meaning every document in the collection is read. This results in high I/O operations, long execution times, and scalability challenges. High  COLLSCAN should be addressed to enhance performance.
+
+=== "Storage metrics"
+    New metrics for analyzing I/O performance:
+
+    - **Storage Bytes Read**: measures the volume of data read from storage during operations, helping identify I/O-intensive queries.
+    - **Storage Time Reading Micros**: tracks the time spent on storage read operations, which is useful for detecting potential storage bottlenecks.
+
+##### New filters
+
+The QAN dashboard now includes several additional MongoDB-specific filter options:
+
+- **Plan Summary**: filters queries by execution plan type (COLLSCAN, IXSCAN, etc.) to easily identify inefficient full collection scans. 
+- **Application Name**: filters queries by the application that executed them. 
+- **User**: isolates queries by specific database users. 
+- **Database**: focuses on particular database instances. 
+
+    ![New Mongo filters](../images/MongoFiltersQAN.png) 
+
+#### Improved query fingerprint visualization
+MongoDB query fingerprints in QAN now display the actual query structure, including operators and field names, similar to the format used in a MongoDB shell. 
+
+The previous display only showed operation type and collection name, which provided limited context for troubleshooting. 
+
+The new fingerprint format makes query patterns immediately recognizable while still anonymizing sensitive data. 
+
+![New Mongo query fingerprint](../images/Mongo_fingerprint.png)
+
+#### Enhanced query examples visualization
+We've refined the query examples interface in Mongo QAN by removing unnecessary enumeration labels ("1 item", "2 item", etc.) and upgrading the JSON viewer component. 
+
+This creates a cleaner display that emphasizes the actual query content rather than distracting metadata.
```

Comment:
for users that actually wanted that metadata (assuming there are any) can they still find that info elsewhere? if yes, might be worth explaining where it is, if no...ok to remain silent on it since we say "it's removed".

Reply:
No, there are no option to get them.